web/web.go

```diff
@@ -64,6 +64,7 @@ import (
 	"github.com/prometheus/prometheus/util/httputil"
 	api_v1 "github.com/prometheus/prometheus/web/api/v1"
 	api_v2 "github.com/prometheus/prometheus/web/api/v2"
+	web_types "github.com/prometheus/prometheus/web/types"
```

Comment:
This imported name is a bit ugly, could you change the directory name?

Reply:
Change the directory name to `web_types`?